kong/plugins/prometheus/handler.lua

```diff
@@ -21,7 +21,7 @@ local PrometheusHandler = {
 }
 
 
-function PrometheusHandler:new()
+function PrometheusHandler:init_worker()
   return prometheus.init()
```

Comment:
Better remove the `return` statement as well. Plugins' `init_worker` don't return anything to the core.

Also, and more importantly, I suggest not invoking this from `init_worker`, but keep it in `init` instead, given it sets values in shms. On an unrelated note, the safeguards in the constructors for this plugin and its dependencies (e.g. on missing shms) don't seem very robust... But that is outside the scope of this PR.

Reply:
Updated. Also prepared for a new release.

Reply:
Actually, @thibaultcha, is the `init` phase at all available for plugins?

Reply:
@gszr Yes, in the top level chunk of the plugin's `handler.lua`. It is invoked by `require` so that could be an issue in PUC-Rio Lua, but not with LuaJIT's fully resumable VM.

Reply:
Done!